app/views/course/achievements/_user_form.html.slim

```diff
@@ -0,0 +1,24 @@
+= simple_form_for achievement, url: award_course_achievement_path(course, achievement) do |f|
+  h3 =  t('.header')
+  = f.error_notification
+  = f.error :achievement
+
+  div.alert.alert-info
+    = simple_format(t('.notice'))
+
+  div.btn-group.pull-right
+    = link_to(t('.mark_all'), '#', class: ['btn', 'btn-primary'], id: ['toggle-on-button'])
```

Comment:
usually how this is done is having a checkbox at the top of the table, above all the checkboxes.